integration_test/README.md

```diff
@@ -91,6 +91,17 @@ make third_party_apps_test PROJECT=${PROJECT} TRANSFERS_BUCKET=${TRANSFERS_BUCKE
 
 As above, you can supply `AGENT_PACKAGES_IN_GCS` or `REPO_SUFFIX` to test a pre-built agent.
 
+Additionally, to run specific third party applications you can use the command:
+
+```
+go test -v ./integration_test/third_party_apps_test.go \
+    -tags=integration_test \
+    -test.run="TestThirdPartyApps/ubuntu-2304-amd64/(nvml|dcgm)"
```

Comment:
Maybe use
```
-test.run="TestThirdPartyApps/.*/(nvml|dcgm)"
```
in the example so that it's agnostic to the platform? (You still need to set `PLATFORMS` as well)